content/events/cshl-biology-of-genomes-2023.md

```diff
@@ -0,0 +1,37 @@
+---
+author: "AnVIL"
+conference: "CSHL BoG"
+description: "The 2023 meeting will address DNA sequence variation and its role in molecular evolution, population genetics and complex diseases, comparative genomics, large-scale studies of gene and protein expression, and genomic approaches to ecological systems."
+eventType: "Conference"
+featured: true
+location: "Cold Spring harbor Laboratory, NY, USA"
+sessions: [{sessionStart: "9 May 2023"}, {sessionStart: "10 May 2023"}, {sessionStart: "11 May 2023"}, {sessionStart: "12 May 2023"}, {sessionStart: "13 May 2023"}]
+timezone: "America/New_York"
+title: "CSHL Biology of Genomes 2023"
+---
+
+<event-hero></event-hero>
+
+### Description
+
+The 2023 meeting will address DNA sequence variation and its role in molecular evolution, population genetics and complex diseases, comparative genomics, large-scale studies of gene and protein expression, and genomic approaches to ecological systems. Both technologies and applications will be emphasized. In addition, there will be a special session on the ethical, legal and social implications (ELSI) of genome research.
+
+The [James P. Taylor Foundation for Open Science](https://jxtxfoundation.org/) will award six scholarships covering full conference registration to outstanding graduate students. Application deadline is February 6, 2023. For details and application instructions, visit [here](https://jxtxfoundation.org/news/2022-12-16-bg/).
+
+Topics:
+Population Genomics
+Systems Genetics
+Evolutionary & Non-human Genomics
+Complex Traits & Microbiome
+Functional Genomics & Epigenetics
+Computational Genomics
+Cancer & Medical Genomics
+Developmental & Single Cell Genomics
+
+### Prerequisites (if any)
```

Comment:
```suggestion
### Prerequisites
```

Reply:
The `(if any)` is probably a field for the event form ... we can just leave this title as `Prerequisites`.